openedx/core/djangoapps/schedules/admin.py

```diff
@@ -122,29 +122,18 @@ def choices(self, changelist):
 
 @admin.register(models.Schedule)
 class ScheduleAdmin(admin.ModelAdmin):
-    list_display = ('username', 'course_id', 'active', 'start_date', 'upgrade_deadline', 'experience_display')
+    list_display = ('username', 'course_id', 'start_date', 'upgrade_deadline', 'experience_display')
     list_display_links = ('start_date', 'upgrade_deadline', 'experience_display')
     list_filter = (
         CourseIdFilter,
         'experience__experience_type',
-        'active',
         KnownErrorCases
     )
     raw_id_fields = ('enrollment',)
     readonly_fields = ('modified',)
     search_fields = ('enrollment__user__username',)
     inlines = (ScheduleExperienceAdminInline,)
-    actions = ['deactivate_schedules', 'activate_schedules'] + experience_actions
```

Comment:
removing the ability to activate and deactivate schedules. 
per this conversation with Mike https://edx-internal.slack.com/archives/G019J2RMNEA/p1609873154057900
it seemed like as long as it is theoretically possible to deactivate enrollments then we don't need this functionality